lib/OpenQA/Scheduler.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015 SUSE Linux GmbH
+# Copyright (C) 2015 SUSE LLC
```

Comment:
In the mail from legal they suggest not to overwrite/delete old (c) rather just append new one.